src/openzaak/setup.py

```diff
@@ -34,6 +34,14 @@ def setup_env():
 
 
 def load_self_signed_certs() -> None:
+    needs_extra_verify_certs = os.environ.get(EXTRA_CERTS_ENVVAR)
```

Comment:
Could you please add the docstring about this change? That you are making `_load_self_signed_certs` idempotent and env var `REQUESTS_CA_BUNDLE` is set inside this function?

Reply:
done!